platforms/enterprise/enterprise/src/integTest/groovy/org/gradle/internal/enterprise/core/BuildScanAutoApplyIntegrationTest.groovy

```diff
@@ -322,6 +324,82 @@ class BuildScanAutoApplyIntegrationTest extends AbstractIntegrationSpec {
         fixture.issuedNoPluginWarningCount(output, 1)
     }
 
+    def "does not auto-apply plugin when Develocity plugin is applied using plugin ID"() {
```

Comment:
❓ Does it make sense to run the whole test class with the Develocity and the Gradle Enterprise fixture? Aka have two subclasses of the test and run the common ones for both the Develocity and the Gradle Enterprise plugin?

Reply:
I don't think it brings anything, and I doubt it will even work. It would make sense to have two subclasses if the production code sometimes applied the GE plugin and sometimes Develocity. Then we could check that the common functionality works the same way for both plugins. But production code always applies just a single version (i.e. GE now). So, the subclass with Develocity plugin will simply not test what we intend to. I think it is best to keep it as-is and then just flip the plugins once the Develocity plugin becomes the default.

Reply:
Alright, let's keep it like that now.